docs/manage/client-mongodb-ssl.md

```diff
@@ -9,32 +9,20 @@ the MongoDB monitoring service.
 Run this command as root or by using the `sudo` command
 
 ```sh
-pmm-admin add mongodb -- --mongodb.tls
+pmm-admin add mongodb --tls
 ```
 
 **Supported SSL/TLS Parameters**
 
-`--mongodb.tls`
-:   Enable a TLS connection with mongo server
+`--tls`
+: Enable a TLS connection with mongo server
 
-`--mongodb.tls-ca`  (string)
-:   A path to a PEM file that contains the CAs that are trusted for server connections.
+`--server-insecure-tls`
+: Skip PMM Server TLS certificate validation
 
-    *If provided*: MongoDB servers connecting to should present a certificate signed by one of these CAs.
+`--tls-skip-verify`
```

Comment:
I'm not sure that it should be here

Reply:
Are you saying remove the tls-skip-verify option?

Reply:
yes

Reply:
no, it was about the flag above. looks like I placed comment in a wrong place.